opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -111,6 +112,25 @@ func resourceOpennebulaVirtualMachine() *schema.Resource {
 				Computed:    true,
 				Description: "Current LCM state of the VM",
 			},
+			"desired_state": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     false,
+				Description: "The desired state of the VM, which can be running, stopped, suspended, undeployed or poweroff; Do not use together with pending option",
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					requestedValue := v.(string)
+					acceptableValues := []string{"running", "stopped", "suspended", "undeployed", "poweroff"}
```

Comment:
pending is missing in the list as we can expect it

Reply:
Which command would result in pending?